packages/plugin-core/src/utils/files.ts

```diff
@@ -0,0 +1,23 @@
+import textextensionslist from "textextensions";
+import open from "open";
+import { DendronError, ERROR_STATUS } from "@dendronhq/common-all";
+import { Logger } from "../logger";
+
+export const TEXT_EXTENSIONS: ReadonlySet<string> = new Set(
+  textextensionslist.map((s) => s.toLowerCase())
+);
+
+/** Opens the given file with the default app.
+ *
+ * Throws a `DendronError` on a failure. */
+export class PluginFileUtils {
+  static async openWithDefaultApp(filePath: string) {
+    await open(filePath).catch((err) => {
```

Comment:
This is just a very thin wrapper to log a failure to open with the plugin's logger. I don't think it makes sense to move this into `common-server`, because then I'd have to remove the call to logger and I'd just be left with `return open(filePath);`.